src/types.ts

```diff
@@ -82,6 +82,11 @@ export interface NetworkConfiguration {
    * @defaultValue 'post'
    */
   method?: HttpMethod;
+
+  /**
+   * Custom headers for HTTP requests to the Collector.
+   */
+  requestHeaders?: [name: string]
```

Comment:
how about matching (option name, type and defaultValue) [the JavaScript tracker](https://github.com/snowplow/snowplow-javascript-tracker/blob/dc8fa2a1df02e74bbb1ee5c313c2b64907b152ea/libraries/browser-tracker-core/src/tracker/types.ts#L219-L223) here?

Reply:
I changed the type to match the JavaScript tracker but I think we should keep the name `requestHeaders` to match the `NetworkConfiguration` in the iOS tracker. So far the RN configuration exactly matches the mobile trackers configuration and I think we should keep that.